### PR TITLE
Upgrade to Gradle 2.10

### PR DESCRIPTION
**Pull request by markelliot**

### CODE REVIEW

File: src/test/groovy/com/palantir/gradle/docker/DockerComposePluginTests.groovy [hunk 1]
```diff
@@ -65,8 +65,8 @@ class DockerComposePluginTests extends AbstractPluginTest {
         when:
         BuildResult buildResult = with('generateDockerCompose', '--stacktrace').buildAndFail()
         then:
-        buildResult.standardError.contains("Failed to resolve Docker dependencies declared in")
-        buildResult.standardError.contains("{{foo:bar}}")
+        buildResult.output.contains("Failed to resolve Docker dependencies declared in")
```
Comment (uschi2000) [inline src/test/groovy/com/palantir/gradle/docker/DockerComposePluginTests.groovy:68]: I understand that the plugin will still work with <2.10, but the stderr/stdout semantics changed for 2.10, hence this change?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-bin.zip
```
Comment (uschi2000) [inline gradle/wrapper/gradle-wrapper.properties:5]: with the 2.10 upgrade, can you get rid of some of that testClasses business in the tests?

File: src/test/groovy/com/palantir/gradle/docker/DockerComposePluginTests.groovy (hunk 1 above)
Comment (markelliot) [inline src/test/groovy/com/palantir/gradle/docker/DockerComposePluginTests.groovy:68]: right

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (markelliot) [inline gradle/wrapper/gradle-wrapper.properties:5]: there's not a meaningful amount of change, still: https://docs.gradle.org/current/userguide/test_kit.html

**Comment by uschi2000** [comment]
Lgtm

**Pull request merged**